setup.py

```diff
@@ -29,10 +29,10 @@
 # 'Development Status :: 5 - Production/Stable'
 release_status = "Development Status :: 5 - Production/Stable"
 dependencies = [
-    # google-api-core[grpc] 1.17.0 causes problems with stream recovery, thus
-    # it should not be used.
+    # google-api-core[grpc] 1.17.0 up to 1.19.1 causes problems with stream
+    # recovery, thus those versions should not be used.
     # https://github.com/googleapis/python-pubsub/issues/74
-    "google-api-core[grpc] >= 1.14.0, != 1.17.0",
+    "google-api-core[grpc] >= 1.14.0, != 1.17.0, != 1.18.0, != 1.19.0, != 1.19.1",
```

Comment:
Perhaps:
google-api-core[grpc] >= 1.14.0, != 1.17.*, != 1.18.*, != 1.19.0, !=1.19.1
instead? The wildcards guarantee we're not forgetting any minor versions in there.

Reply:
AFAIK releases are only incremental and it's highly unlikely that a minor bugfix version, say, 1.17.1 is released in the future?

Reply:
Yes, but it's also slightly better for docs - you know for sure that there isn't some minor version that's affected. I know we have that in the comment, but comments can change. This is likely going to be around until we make a brand new version of this library.

Reply:
```suggestion
    "google-api-core[grpc] >= 1.14.0, != 1.17.*, != 1.18.*, != 1.19.*",
```

Reply:
Agree, made the list of restricted versions wider, all bugfix versions of problematic minor versions are now excluded, too.

Reply:
Also made a quick test, seems like the pins are correct:
```py
In [40]: import pkg_resources
In [41]: versions = ['1.16.0', '1.17.0', '1.18.0', '1.18.3', '1.19.0', '1.19.1', '1.20.0']
In [42]: requirement = pkg_resources.Requirement.parse("google-api-core[grpc] >= 1.14.0, != 1.17.*, != 1.18.*, != 1.19.*")
In [43]: list(requirement.specifier.filter(versions))
Out[43]: ['1.16.0', '1.20.0']

```